tests/TestCase.php

```diff
@@ -18,6 +18,8 @@ class TestCase extends \PHPUnit_Framework_TestCase
 
     public function __construct()
     {
+        date_default_timezone_set('Europe/Amsterdam');
```

Comment:
$this->setDefaultsDuringDevelopment(); at the bottom of this constructor does the same, doesnt't it? If it's not working then maybe that should be called first

Reply:
yes will remove this.